app/retail/templates/emails/new_kudos.html

```diff
@@ -60,72 +60,128 @@
       flex-direction: column;
     }
   }
+  
+  .kudos-fancy-bg {
```

Comment:
Can you fix the indentation throughout this file?

Reply:
yes sir, just wanted to make sure the basic template works before I do any clean up work. :-)

Reply:
please check again.